src/include/function/pointer_function_executor.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "common/vector/value_vector.h"
+
+namespace kuzu {
+namespace main {
+class ClientContext;
+}
+
+namespace function {
+
+struct PointerFunctionExecutor {
```

Comment:
This executor doesn't take in any parameter. So I assume ConstFunctionExecutor is a better naming.